### PR TITLE
Change autocomplete to be overlay

### PR DESCRIPTION
**Pull request by JohnPadgettCivica**
Change to make the auto complete overlay

### CODE REVIEW

File: src/EA.Weee.Web/Scripts/weee-application.js [hunk 1]
```diff
@@ -171,6 +171,7 @@
             //source: suggest,
             element: newElement,
             name: element.id + "-auto",
+            displayMenu: 'overlay',
```
Comment (meglenapnovoselska) [inline src/EA.Weee.Web/Scripts/weee-application.js:174]: is that for the drop downs UX bug, raised by Sarah?

**Review by meglenapnovoselska — APPROVED** [review]

File: src/EA.Weee.Web/Scripts/weee-application.js (hunk 1 above)
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Scripts/weee-application.js:174]: Yeah it is, thankfully there was an option on the control :)

File: src/EA.Weee.Web/Scripts/weee-application.js [hunk 2]
```diff
@@ -155,7 +155,8 @@
                 break;
             }
         }
-
+        var useOverLay = element.classList.contains("use-overlay");
+        console.log(useOverLay);
```
Comment (meglenapnovoselska) [inline src/EA.Weee.Web/Scripts/weee-application.js:159]: can you remove it?

**Review by meglenapnovoselska — APPROVED** [review]

File: src/EA.Weee.Web/Scripts/weee-application.js (hunk 2 above)
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web/Scripts/weee-application.js:159]: whoops

**Pull request merged**